pom.xml

```diff
@@ -253,6 +297,7 @@
                     <dryRun>false</dryRun>
                     <excludes>
                         <exclude>plugin/org/itsallcode/openfasttrace/maven/HelpMojo.java</exclude>
+                        <exclude>test/resources/*/src/**</exclude>
```

Comment:
I have added this, but header is available in some of previous tests. Please let me know if I should remove from them also.

Reply:
Thank you, that's fine. We can do this later!